frontend/packages/topology/src/actions/contextMenuActions.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Node } from '@patternfly/react-topology';
+import { ActionsLoader } from '@console/shared';
+import { createContextMenuItems } from '../components/graph-view';
+
+export const contextMenuActions = (element: Node): React.ReactElement[] => {
+  return [
+    <ActionsLoader
+      key={`topology-actions-${element.getType()}`}
```

Comment:
`key` here can be a constant because there's no conflicts
`topology` is sufficient